### ISSUE TITLE
Failed to create cluster using Libvirt IPI

### ISSUE THREAD

**Issue by juliobarreto** [openshift/installer#4108]
# Version

```console
$ openshift-install version
release image registry.svc.ci.openshift.org/origin/release:4.5
```

# Platform:
libvirt

* IPI (automated install with `openshift-install`)

# What happened?

I follow all the steps described in "https://github.com/openshift/installer/blob/master/docs/dev/libvirt/README.md".

Then he downloads the RHCOS image, creates the bootstrap and master-0 machine, then the following errors occur:

INFO Waiting up to 20m0s for the Kubernetes API at https: //api.ocp.example.net: 6443 ...
ERROR Attempted to gather ClusterOperator status after installation failure: listing ClusterOperator objects: Get "https://api.ocp.example.net:6443/apis/config.openshift.io/v1/clusteroperators": dial tcp 192.168.126.11:6443 : connect: connection refused
INFO Pulling debug logs from the bootstrap machine
INFO Bootstrap gather logs captured here "/root/ocp/installer/test/log-bundle-20200828001915.tar.gz"
FATAL Bootstrap failed to complete: failed waiting for Kubernetes API: Get "https://api.ocp.example.net:6443/version?timeout=32s": dial tcp 192.168.126.10:6443: connect: connection refused

# What you expected to happen?

I hope that the servers are successfully created and the Openshift cluster is provisioned

# How to reproduce it (as minimally and precisely as possible)?

```console
[root@p50 installer]#  bin/openshift-install create cluster --dir test
? SSH Public Key /root/.ssh/id_rsa.pub
? Platform libvirt
? Libvirt Connection URI qemu+tcp://192.168.122.1/system
? Base Domain example.net
? Cluster Name ocp
? Pull Secret [? for help] *************
INFO Obtaining RHCOS image file from 'https://releases-art-rhcos.svc.ci.openshift.org/art/storage/releases/rhcos-4.6/46.82.202008260918-0/x86_64/rhcos-46.82.202008260918-0-qemu.x86_64.qcow2.gz?sha256=c7c8ce8116e16a3cb4b9c197320470fc0e287bd69cc6a7186872ec203dc6096a' 
INFO The file was found in cache: /root/.cache/openshift-installer/image_cache/a4a1cdd5835e20adabf84755c58f94e4. Reusing... 
INFO Creating infrastructure resources...         
INFO Waiting up to 20m0s for the Kubernetes API at https://api.ocp.example.net:6443... 
ERROR Attempted to gather ClusterOperator status after installation failure: listing ClusterOperator objects: Get "https://api.ocp.example.net:6443/apis/config.openshift.io/v1/clusteroperators": dial tcp 192.168.126.11:6443: connect: connection refused 
INFO Pulling debug logs from the bootstrap machine 
INFO Bootstrap gather logs captured here "/root/ocp/installer/test/log-bundle-20200828001915.tar.gz" 
FATAL Bootstrap failed to complete: failed waiting for Kubernetes API: Get "https://api.ocp.example.net:6443/version?timeout=32s": dial tcp 192.168.126.10:6443: connect: connection refused 

```

# Anything else we need to know?

Instead of using Dnsmasq can I use another type of DNS? How would the entries be?

# References

**Comment by cfergeau** [comment]
Could you check in virt-manager whether the VMs are up, and what state they are in?

**Comment by cloudxabide** [comment]
i am running in to a (potentially) similar issue - and I have some additional data to provide.
TL;DR: my master node(s) go from running RHCOS to Fedora 33, and the ovs package is removed.

`[root@borealis ~]# ~/OCP4/installer-2021-01-06/bin/openshift-install version
/root/OCP4/installer-2021-01-06/bin/openshift-install unreleased-master-4117-g4cc5df0f7c9a5e9f89ce124bc2a95469d4db39f0
built from commit 4cc5df0f7c9a5e9f89ce124bc2a95469d4db39f0
release image registry.svc.ci.openshift.org/origin/release:4.7
[root@borealis ~]# virsh list --all
 Id   Name                     State
----------------------------------------
 1    laptop-qhknd-master-0    running
 2    laptop-qhknd-bootstrap   running

[root@borealis ~]# virsh net-list
 Name           State    Autostart   Persistent
-------------------------------------------------
 default        active   yes         yes
 laptop-qhknd   active   yes         yes

=====================================================
Follow the timeline, but.. seriously... huh?

 22:58:16 up 13 min,  0 users,  load average: 0.11, 0.07, 0.02
Red Hat Enterprise Linux CoreOS release 4.7
NetworkManager-ovs-1.26.0-9.el8_3.x86_64

 22:58:26 up 13 min,  0 users,  load average: 1.05, 0.26, 0.08
Red Hat Enterprise Linux CoreOS release 4.7
NetworkManager-ovs-1.26.0-9.el8_3.x86_64

 22:58:52 up 0 min,  0 users,  load average: 0.52, 0.11, 0.04
Fedora release 33 (Thirty Three)
warning: Found bdb Packages database while attempting sqlite backend: using bdb backend.
`
